.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "globals": {
+        "Atomics": "readonly",
```

Comment:
out of curiosity, is there a reason for these global rules?

Reply:
I used `eslint --init` and it added these. I'm not sure what they do but I'll review. Thanks for calling this out.